--- a/far2l/src/cfg/ConfigOpt.cpp
+++ b/far2l/src/cfg/ConfigOpt.cpp
@@ -180,7 +180,7 @@ const ConfigOpt g_cfg_opts[] {
 	{OST_COMMON, NSecInterface, "TimeSeparator", &Opt.strTimeSeparator, GetTimeSeparatorDefaultStr()},
 	{OST_COMMON, NSecInterface, "DecimalSeparator", &Opt.strDecimalSeparator, GetDecimalSeparatorDefaultStr()},
 
-	{OST_COMMON, NSecInterface, "OSC52ClipSet", &Opt.OSC52ClipSet, 0},
+	{OST_COMMON, NSecInterface, "OSC52ClipSet", &Opt.OSC52ClipSet, 1},
 	{OST_COMMON, NSecInterface, "TTYPaletteOverride", &Opt.TTYPaletteOverride, 1},
 
 	{OST_NONE,   NSecInterface, "ShowTimeoutDelFiles", &Opt.ShowTimeoutDelFiles, 50},
@@ -278,7 +278,7 @@ const ConfigOpt g_cfg_opts[] {
 	{OST_COMMON, NSecSystem, "SaveViewHistory", &Opt.SaveViewHistory, 1},
 	{OST_COMMON, NSecSystem, "HistoryRemoveDupsRule", &Opt.HistoryRemoveDupsRule, HISTORY_REMOVE_DUPS_BY_NAME_EXTRA},
 	{OST_COMMON, NSecSystem, "AutoHighlightHistory", &Opt.AutoHighlightHistory, 1},
-	{OST_COMMON, NSecSystem, NParamAutoSaveSetup, &Opt.AutoSaveSetup, 0},
+	{OST_COMMON, NSecSystem, NParamAutoSaveSetup, &Opt.AutoSaveSetup, 1},
 	{OST_COMMON, NSecSystem, NParamAutoSavePanels, &Opt.AutoSavePanels, 0},
 	{OST_COMMON, NSecSystem, "DeleteToRecycleBin", &Opt.DeleteToRecycleBin, 0},
 	{OST_COMMON, NSecSystem, "DeleteToRecycleBinKillLink", &Opt.DeleteToRecycleBinKillLink, 1},
